packages/report-server/src/reportBuilder/fetch/functions/dataGroup.ts

```diff
@@ -7,22 +7,33 @@ import { Aggregator } from '../../../aggregator';
 import { FetchReportQuery } from '../../../types';
 import { FetchResponse } from '../types';
 
+type DataGroupParams = {
+  dataGroups: unknown;
+  dataElements: undefined;
```

Comment:
```suggestion
  dataElements: unknown;
```
I think `unknown` is the super type here, we don't technically know that the field is undefined just as yet 👍